packages/side-nav/src/vaadin-side-nav-item.d.ts

```diff
@@ -100,6 +100,11 @@ declare class SideNavItem extends SideNavChildrenMixin(DisabledMixin(ElementMixi
    */
   readonly current: boolean;
 
+  /**
+   * The target of the link. Works only when `path` is set.
+   */
+  target: string | null | undefined;
```

Comment:
Should there be a type here that only accepts something like `'_self' | '_blank' | '_parent' | '_top'`?

Reply:
`target` in `HTMLAnchoElement` doesn't seem to get any special type ([source](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.htmlanchorelement.html#target))

Reply:
A niche use-case would be to configure a specific browsing context, which could have any name apparently (https://github.com/vaadin/flow-components/issues/5091#issuecomment-1860055268). So the union type would need to include `string` as well. But just starting with `string` is fine IMO 👍